shakenfist/net.py

```diff
@@ -321,7 +321,7 @@ def create(self):
             self.update_dhcp()
         else:
             db.enqueue('networknode', DeployNetworkTask(self.uuid))
-            db.add_event('network', self.uuid, 'deploy', 'enqueued')
+            self.add_event('network', self.uuid, 'deploy', 'enqueued')
```

Comment:
I think this correction would be better as `self.add_event('deploy', 'enqueued')`

Reply:
Oh yeah. Fair.